addons/purchase/report/purchase_order_templates.xml

```diff
@@ -37,22 +37,26 @@
                 </div>
                 <div t-if="o.state in ['purchase','done'] and o.date_approve" class="col-3 bm-2">
                     <strong>Order Date:</strong>
-                    <p t-field="o.date_approve" class="m-0"/>
+                    <p t-esc="o.date_approve.strftime('%m/%d/%Y')" class="m-0"/>
```

Comment:
```suggestion
                    <p t-field="o.date_approve" t-options="{'date_only': 'true'} class="m-0"/>
```
the date format should depends on the language not be hardcoded

Reply:
@Whenrow 
Thanks for the suggestion, I have done the suggested changes.